src/server/handlers/github-auth.js

```diff
@@ -66,7 +66,7 @@ export const verify = (req, res, next) => {
     logger.info('User successfully authenticated');
 
     // Redirect to logged in URL
-    res.redirect('/dashboard');
+    res.redirect('/dashboard/select-repositories');
```

Comment:
After logging in, redirect to the dashboard select-repositories view as if logging in for the first time.